EIPS/payoutable-token-draft.md

```diff
@@ -0,0 +1,302 @@
+---
+eip: <to be assigned>
+title: Payoutable Token
+author: Fernando Paris <fer@io.builders>
+status: Draft
+type: Standards Track
+category:  ERC
+created: 2019-05-10
+requires: 20
+
+---
+
+## Simple Summary
+An extension to the ERC-20 standard token that allows Token wallet owners to request payut from their wallet, by calling the smart contract and attaching a payout instruction string.
+
+## Actors
+
+#### Token Wallet Owners
+The person or company who owns the wallet, and will order payout.
+
+#### Token contract operator
+The entity, company responsible/owner of the token contract, and token issuing/minting. This actor is in charge of trying to fullfill all payout request(s), reading the payout instruction(s), and corelate the payout details.
+
+#### Orderer
+An actor who is enable to initiate payout orders on behalf ot a token wallet owner.
+
+
+## Abstract
+Token wallet owners (or approved addresses) can order payout requests through  blockchain. This is done by calling the ```orderPayoutFrom``` or ```orderPayoutFrom``` methods, which initiate the workflow for the token contract operator to either honor or reject the payout request. In this case, payout instructions are provided when submitting the request, which are used by the operator to determine the destination of the funds.
+
+In general, it is not advisable to place explicit routing instructions for the payouts on a verbatim basis on the blockchain, and it is advised to use a private communication alternatives. such as private channels, encrypted storage or similar,  to do so (external to the blockchain ledger). Another (less desirable) possibility is to place these instructions on the instructions field on encrypted form.
```

Comment:
There has to be a coma before `such as`.
I think it has to be `..field in encrypted form`